google/import_google_service_account_test.go

```diff
@@ -0,0 +1,76 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"os"
+)
+
+func TestAccGoogleServiceAccount_importBasic(t *testing.T) {
+	t.Parallel()
+
+	resourceName := "google_service_account.acceptance"
+	sa_name := "terraform-" + acctest.RandString(10)
+	conf := testAccGoogleServiceAccount_import(sa_name)
```

Comment:
nit: s/sa_name/saName/

Also nit: this is (and other variables defined here are) only used in one place, so you could pretty easily inline them

Reply:
Done.